tensorflow_addons/losses/triplet.py

```diff
@@ -118,12 +118,12 @@ def custom_distance(batch):
 
     # Build pairwise squared distance matrix
```

Comment:
Not sure how relevant this comment anymore. It's also hanging with an empty line, next to the line with `squared=False`